### ISSUE TITLE
aggregateWindow + cumulativeSum not producing correct results.

### ISSUE THREAD

**Issue by russorat** [influxdata/flux#2446]
could be related to https://github.com/influxdata/flux/issues/2425

Recently, i noticed my graphs using `cumulativeSum` were showing flat lines after a few calculations:
![image](https://user-images.githubusercontent.com/4805997/73583051-f9f0e380-4444-11ea-8c39-b42f165be690.png)

Looking at the raw data from one repo, i could see that i had events happening towards the end of the month, but they were not getting included in my cumulative total:
![image](https://user-images.githubusercontent.com/4805997/73583131-5bb14d80-4445-11ea-8be9-145e79caaa64.png)

My original query used:
```
 |> aggregateWindow(every: 1d, fn: count)
 |> cumulativeSum(columns: ["_value"])
```

In order to work around this, I updated the query to be the following:

![image](https://user-images.githubusercontent.com/4805997/73582995-b1d1c100-4444-11ea-921a-7cccc5d07272.png)

```
  |> window(every: 1d)
  |> count(column: "_value")
  |> group(columns: ["repository"])
  |> cumulativeSum(columns: ["_value"])  
```

**Comment by mark-rushakoff** [comment]
We believe that was a bug in v0.59.4 (introduced in #2312) which was fixed in v0.59.5 (with #2431).

**Comment by jsternberg** [comment]
I'd be interested in more information about this and hopefully a reproducer. `cumulativeSum` doesn't have the same problem that caused the problem with derivative and I am not able to reproduce the issue when testing on v0.59.4. The revert is likely to fix it just because it reverts to the old implementation, but I would like to know what is causing the issue as the new implementation is faster and we want to switch to it at some point.
